<?xml version="1.0"?>
<robot name="simple_chair" xmlns:xacro="http://wiki.ros.org/xacro">
  
  <xacro:arg name="prefix" default="chair_"/>
  <xacro:arg name="connected_to" default=""/>
  <xacro:arg name="xyz" default="'0 0 0'"/>
  <xacro:arg name="rpy" default="'0 0 0'"/>
  <xacro:arg name="height" default="1.0"/>
  <xacro:arg name="x_size" default="0.5"/>
  <xacro:arg name="y_size" default="0.5"/>
  <xacro:arg name="wall_thickness" default="0.02"/>
  <xacro:arg name="mass" default="3.0"/>

  <xacro:include filename="$(find autonomous_furniture)/simple_chair.xacro"/>

  <xacro:simple_chair prefix="$(arg prefix)" connected_to="$(arg connected_to)" xyz="$(arg xyz)" rpy="$(arg rpy)"
                      height="$(arg height)" x_size="$(arg x_size)" y_size="$(arg y_size)" thickness="$(arg wall_thickness)"
                      mass="$(arg mass)"/>

</robot>
